functorch/test/test_aotdispatch.py

```diff
@@ -689,6 +689,7 @@ def assert_compiler(gm: torch.fx.GraphModule, _):
 # entries in here don't work and need to be fixed.
 # Each one of these is a bug (or needs to be investigated)
 aot_autograd_failures = {
+    xfail('__getitem__', ''), # new failure? needs debug
```

Comment:
@albanD is working on this, it's due to lack of autograd support for monkeypatch